backend/bamboo/forms/media.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
```

Comment:
There is no need to define forms. You could define a schema for the input file, and then validate the file with [`FileSize` and `FileType` validators](https://apiflask.com/api/fields/).

Example application: https://github.com/apiflask/apiflask/blob/main/examples/file_upload/app.py